spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.describe "Customer Show", type: :feature do
+  describe "as a visitor" do
+    it "shows the customers items, and items' attritutes" do
+      abel = Customer.create!(name: "Abel")
+
+      albertsons = Supermarket.create!(name: "Albertsons", location: "Denver")
+
+      apple = albertsons.items.create!(name: "apple", price: 1)
+      bread = albertsons.items.create!(name: "bread", price: 2)
+      corn = albertsons.items.create!(name: "corn", price: 4)
+
+     CustomerItem.create!(customer: abel, item: apple)
+     CustomerItem.create!(customer: abel, item: bread)
+     CustomerItem.create!(customer: abel, item: corn)
+
+      visit "customers/#{abel.id}"
+
+      expect(page).to have_content("Abel")
```

Comment:
We'd love to see all these assertions in within blocks. Just asserting that a `$2` is not very meaningful unless we know it's in the section for bread. We can also get false positives on these tests is different items have the same price, or if there is a section on the page with other numbers displayed. Alternatively, if you set a text label up in your view, you can also anchor these assertions with their label like so: `expect(page).to have_content("Price: 2")`